src/storage/DataAccessorBasedStore.ts

```diff
@@ -185,7 +185,7 @@ export class DataAccessorBasedStore implements ResourceStore {
     try {
       return await this.accessor.getMetadata(identifier);
     } catch (error: unknown) {
-      if (error instanceof NotFoundHttpError) {
+      if (error instanceof NotFoundHttpError && !this.identifierStrategy.isRootContainer(identifier)) {
```

Comment:
Comment of why this is happening?

Reply:
I'm going to add a comment summarizing this in the code, but to fully explain what happened:

Solid says to make sure we don't have resources both with and without the slash, which is why this function exists. But removing a slash has a specific meaning in Solid. If, for example, our resource is `/foo/bar/`. Removing the trailing slash means going to the parent container `/foo/` and accessing the resource with the slug `bar` there. But in the case of root containers there is no parent container so it is undefined what happens when trying to do that. It's also simply an optimization since we know for sure the version without a slash can't exist (unless there is a problem with the `IdentifierStrategy`).